woocommerce/woocommerce
# delete vs trash product in e2e api

### Changes proposed in this Pull Request:

This PR changes the api package to delete instead of trashing products. The products that are currently trashed via API in the E2E tests are

- Variable Product with Three Attributes
- Grouped Product with Three Children
- Add-on B
- Add-on A
- Base Unit
- External product

The grouped product test wasn't deleting the simple products created during the test. This PR includes changing the test to delete them.

### How to test the changes in this Pull Request:

1. Checkout the latest `trunk` branch
2. Run `npm run build:packages`
3. Checkout this branch
4. Run e2e tests
5. Check that trash products contain 1,2,6 the above list. 3-5 should be in the published list
6. Empty the Products Trash
7. Run `npm run build:packages`
8. Run `npx wc-e2e tests/e2e/specs/rest-api/api.test.js`
9. Check that neither published nor trash products contain the above list

### Other information:


### Changelog entry

N/A
